### ISSUE TITLE
Failure when experimenting with Chat

### ISSUE THREAD

**Issue by farbere** [WolframResearch/Chatbook#269]
I just tried to use the new AI Chat feature in Mathematica. Executing any cells brings up errors: even saying "Hello, chat" resulted in a Failure. See the attached screenshot.

<img width="920" alt="Screen Shot 2023-07-20 at 1 51 39 PM" src="https://github.com/WolframResearch/Chatbook/assets/89481346/994ee057-5623-4cc3-b325-7b78ae7a25b0">

<details>
<summary>Debug Data</summary>

| Property | Value |
| --- | --- |
| Name | ``"Wolfram/Chatbook"`` |
| Version | ``"1.0.3"`` |
| EvaluationEnvironment | ``"Session"`` |
| FrontEndVersion | ``"Desktop: 13.2 for Mac OS X x86 (64-bit) (November 18, 2022)"`` |
| KernelVersion | ``"13.2.0 for Mac OS X x86 (64-bit) (November 18, 2022)"`` |
| SystemID | ``"MacOSX-x86-64"`` |
| Notebooks | ``True`` |
| DynamicEvaluation | ``False`` |
| SynchronousEvaluation | ``False`` |
| TaskEvaluation | ``False`` |

## Settings

| Property | Value |
| --- | --- |
| Assistance | ``Automatic`` |
| AutoFormat | ``True`` |
| BasePrompt | ``Automatic`` |
| ChatContextCellProcessingFunction | ``Automatic`` |
| ChatContextPostEvaluationFunction | ``Automatic`` |
| ChatContextPreprompt | ``Automatic`` |
| ChatDrivenNotebook | ``False`` |
| ChatHistoryLength | ``25`` |
| DynamicAutoFormat | ``Automatic`` |
| FrequencyPenalty | ``0.1`` |
| IncludeHistory | ``Automatic`` |
| LLMEvaluator | ``<|"Icon" -> RawBoxes[TemplateBox[{}, "ChatIconCodeAssistant"]], "Origin" -> "Wolfram/Chatbook", "PacletName" -> "Wolfram/Chatbook", "Post" -> Missing["NotAvailable", "/Users/FarberE/Library/Mathematica/Paclets/Repository/Wolfram__Chatbook-1.0.3/LLMConfiguration/Personas/CodeAssistant/Post.md"], "Pre" -> "You are a helpful Wolfram Language assistant named Code Assistant. Your job is to do the following:\n\n* Offer Wolfram Language suggestions based on previous inputs.\n* Help fix errors.\n* Expla...`` |
| MaxTokens | ``Automatic`` |
| MergeMessages | ``True`` |
| Model | ``"gpt-3.5-turbo"`` |
| PresencePenalty | ``0.1`` |
| ShowMinimized | ``Automatic`` |
| StreamingOutputMethod | ``Automatic`` |
| Temperature | ``0.7`` |
| Tools | ``Automatic`` |
| ToolsEnabled | ``False`` |
| TopP | ``1`` |
| Data | ``<|"Messages" -> {<|"role" -> "system", "content" -> "You are a helpful Wolfram Language assistant named Code Assistant. Your job is to do the following:\n\n* Offer Wolfram Language suggestions based on previous inputs.\n* Help fix errors.\n* Explain error messages so the user understands what went wrong.\n* Explain outputs that may be confusing to inexperienced users.\n* Assist the user with other topics as needed. While you specialize in Wolfram Language, you can help with any topic.\n\n# Gener...`` |
| Container | ``Wolfram`Chatbook`Actions`Private`container$8367`` |
| CellObject | ``CellObject["fe2345cd-d867-4e79-ad41-a24baa69432a", "0124fb91-b19c-4788-abf1-6a754cd63553"]`` |
| Task | ``TaskObject[Association["TaskUUID" -> "bbf6e6cd-87bd-4163-90e2-2ce87cc08f81", "TaskEnvironment" -> "External", "TaskType" -> "Asynchronous", "AsynchronounsTaskID" -> 3, "EvaluationExpression" :> None, HandlerFunctions -> Association[], HandlerFunctionsKeys -> {"BodyChunk", "StatusCode", "Task", "TaskStatus", "EventName"}, AutoRemove -> True]]`` |

## Stack Data
```
catchTopAs
throwFailure
trimURL
```

## Failure Data

```
<|
  "Evaluation" :> Wolfram`Chatbook`Actions`Private`writ
```

</details>

**Comment by raverydavis** [comment]
Hi, that Failure object response is a known bug that we're hoping is fixed in paclet version 1.0.4 - there's a pre-release available right now if you'd like to test it out!
```
PacletInstall["https://github.com/WolframResearch/Chatbook/releases/download/v1.0.4/Wolfram__Chatbook-1.0.4.paclet", 
 ForceVersionInstall -> True]
```
Should install it. If you still see any Failure objects, let us know!

For the MetaInformation piece, there was a paclet updated to support some of the LLM functionality we added.
```
PacletInstall["Templating", UpdatePacletSites -> True]
```
should update that paclet so you see the proper formatting of prompts and personas from the Prompt Repository!

**Comment by farbere** [comment]
When I call the first PacletInstall, I receive the following message repeatedly: 

ChannelObject::connfail: Could not connect to the server "mqtts://channelbroker-mqtt.wolframcloud.com:8883/".
